presto-docs/src/main/sphinx/security/ldap.rst

```diff
@@ -0,0 +1,220 @@
+
+======================================================= ======================================================
+Property                                                Description
+======================================================= ======================================================
+``authentication.ldap.enabled``                         Enable LDAP authentication for the Presto coordinator.
+                                                        Must be set to ``true``. Default value is ``false``.
+``authentication.ldap.server-type``                     The type of LDAP server implementation.
+                                                        Set to ``active_directory`` or ``openLDAP``.
+``authentication.ldap.url``                             The url to the LDAP server. We recommend using 
+                                                        ``ldaps://`` to enable Secure LDAP over SSL.
+``authentication.ldap.ad-domain``                       The domain name of the Active Directory server.
+                                                        This property is optional if 
+                                                        ``authentication.ldap.server-type`` is set to
+                                                        ``openLDAP``. Eg: ``corp.domain.com``.
+``authentication.ldap.base-dn``                         The base LDAP distinguished name for the user who
+                                                        who tries to connect to the server. This property is 
+                                                        optional if ``authentication.ldap.server-type`` is set
+                                                        to ``active_directory``, but mandatory if
+                                                        ``authentication.ldap.group-dn`` is set.
+``http-server.https.enabled``                           Enables HTTPS access for the Presto coordinator.
+                                                        Should be set to ``true``. Default value is 
+                                                        ``false``.
+``http-server.https.port``                              HTTPS server port.
+``http-server.https.keystore.path``                     The location of the Java Keystore file that will be
+                                                        used to secure TLS.
+``http-server.https.keystore.key``                      The password for the keystore. This must match the
+                                                        password you specified when creating the keystore.
+======================================================= ======================================================
+
+Authorization based on LDAP Group Membership
+********************************************
+
+Once the user is successfully authenticated via LDAPS, the user can be 
+validated to determine if that user belongs to a specified group. This feature
```

Comment:
Perhaps rephrase this as: "You can further restrict the set of users allowed to connect to Presto coordinator based on their group memberships."